plugins/samson_ledger/test/lib/samson_ledger/client_test.rb

```diff
@@ -29,10 +40,71 @@
   end
 
   describe ".post_deployment" do
+    before do
+      @asserts = Proc.new {}
```

Comment:
I can't really do `let(:calls) { [] }`. It throws random `[BUG] Segmentation fault at 0x00000000000000` error. Suspecting there might be a bug in the `single_cov` gem because the random error went away after I removed `require 'single_cov'` from `test_helper.rb`.

So instead I do `@results = []` here.